public/invite.html.erb

```diff
@@ -13,6 +13,7 @@
   </p>
   <p>
     <h1 id='team_caption'>Get Invited to Team <u><span id='team_name'></span></u></h1>
+    <span>If you have previously joined this team, click <a id='team_link'>here</a> to be redirected</span>
```

Comment:
To be super nitpicky, let's call this `team_workspace_url` to be consistent with the other field names.

Add a period since it's a full sentence.